src/javascript/config.js

```diff
@@ -24,12 +24,16 @@ const isProduction = () => {
 
 const binary_desktop_app_id = 14473;
 
-const getAppId = () => {
+const getAppId = (domain) => {
     let app_id = null;
     const user_app_id   = ''; // you can insert Application ID of your registered application here
     const config_app_id = window.localStorage.getItem('config.app_id');
     const is_new_app    = /\/app\//.test(window.location.pathname);
-    if (config_app_id) {
+    const domain_app_id = domain_app_ids[domain];
+
+    if (domain_app_id) {
+        app_id = domain_app_id;
+    } else if (config_app_id) {
```

Comment:
config_app_id should be the first if, because we always want the localstorage to override any default settings.